sample/src/main/java/com/smileidentity/sample/viewmodel/MainScreenViewModel.kt

```diff
@@ -160,8 +163,8 @@ class MainScreenViewModel : ViewModel() {
      * will automatically pick up the correct environment)
      */
     fun toggleEnvironment() {
-        pendingJobCountJob.cancel()
-        backgroundJobsPollingJob.cancel()
+        pendingJobCountJob?.cancel()
+        backgroundJobsPollingJob?.cancel()
```

Comment:
These are redundant, they are never null
```suggestion
        pendingJobCountJob.cancel()
        backgroundJobsPollingJob.cancel()
```